clustermesh-apiserver/README.md

```diff
@@ -1,29 +1,85 @@
 # API server for Cilium ClusterMesh
 
+Cilium uses a clustermesh-apiserver in setups that do not use a KVstore as the
+identity allocation mode but use a CRD mode (e.g. external workloads).
+
+Since etcd is used in a clustermesh for data synchronization an etcd server
+is deployed along with clustermesh-apiserver.
+
+The API server itself performs the following operations:
+
+### K8s synchronization
+
+It performs the job of synchronizing CiliumIdentites, CiliumEndpoints,
+CiliumNodes and Kubernetes services from k8s datastore to the KVStore (etcd).
+
+When used in an External Workloads setup it also creates a CiliumNode resource
+for each workload name and allocates its identity.
```

Comment:
"also creates a CiliumNode resource"-> "also creates a CiliumNode and CiliumEndpoint resources"